std_msgs/Header header
float64 simtime
mujoco_ros/BodyState[] states